sbncode/FlashMatch/FlashPredict_module.cc

```diff
@@ -1810,10 +1892,10 @@ double FlashPredict::wallXWithMaxPE(const OpHitIt opH_beg,
 
 
 double FlashPredict::fractTimeWithFractionOfLight(
-  const SimpleFlash& simpleFlash, const double sum_pe, const double fraction_pe,
+  const std::vector<recob::OpHit>& ophits, const double sum_pe, const double fraction_pe,
   const bool use_square_pe, const bool only_unpe) const
 {
-  std::vector<recob::OpHit> timeSortedOpH(simpleFlash.opH_beg, simpleFlash.opH_end);
+  std::vector<recob::OpHit> timeSortedOpH(ophits.begin(), ophits.end());
```

Comment:
This function should receive an already time sorted `ophits`, so there's no need to sort it again, nor to create another copy.